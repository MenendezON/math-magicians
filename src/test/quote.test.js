import * as React from 'react';
import { render } from '@testing-library/react';
import QuoteComponent from '../pages/Quote';

describe('QuoteDisplay', () => {
  it('renders QuoteDisplay component', () => {
    const { container } = render(<QuoteComponent />);
    expect(container).toMatchSnapshot();
  });
});