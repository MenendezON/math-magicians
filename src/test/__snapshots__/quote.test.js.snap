// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteDisplay renders QuoteDisplay component 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
